package-examples/helloworld-tshirt/README.md

```diff
@@ -0,0 +1,75 @@
+# helloworld-tshirt
+
+This is a package that has both a setter and a simple hydration function.  The
+hydration function examines the configuration and looks for the t-shirt size
+value.  Depending on the size (small, medium, large) it then sets the 
+resource sizes.
+
+## Steps
+
+This is a sample workflow where 
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Configure functions](#configure-functions)
+4. [Render the declared values](#render-the-declared-values)
+5. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld-tshirt
+
+    fetching package /package-examples/helloworld-tshirt from https://github.com/GoogleContainerTools/kpt to helloworld-tshirt
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+  $ kpt pkg tree helloworld-tshirt/
+
+    PKG: helloworld-tshirt
+    ├── [Kptfile]  Kptfile helloworld-tshirt
+    ├── [helloworld.yaml]  Deployment helloworld-gke
+    └── [helloworld.yaml]  Service helloworld-gke
+
+### Configure functions
+
+The package contains a function pipeline in the `Kptfile` which has
+one `apply-setter` and 'example-tshirt' functions.  The `apply-setter` function allows you to set a simple value throughout the package configuration.  In 
```

Comment:
apply-setters.

example-tshirt should be verbatim same as apply-setters

What is goal of `example-tshirt` here?